GVRf/Framework/settings.gradle

```diff
@@ -1,6 +1,12 @@
 include ':framework'
 include ':backend_oculus'
 include ':backend_daydream'
+include ':gvr-simplesample'
```

Comment:
Please do not commit changes to settings.gradle. extra_settings.gradle exists for this very reason - so you can add whatever other projects you want to the framework workspace outside of the source controlled directories.

Reply:
oops.. i forgot to remove this one.